<?xml version="1.0" encoding="UTF-8"?>
<testExecutions version="1">
  <file path="/Users/lenny/Projects/hello-next/src/__tests__/example.test.js">
    <testCase name="sum function should add two numbers together" duration="2"/>
    <testCase name="testing with jest matchers object assignment" duration="1"/>
    <testCase name="testing with jest matchers adding positive numbers is not zero" duration="10"/>
    <testCase name="testing with jest truthiness null" duration="0"/>
    <testCase name="testing with jest truthiness zero" duration="0"/>
    <testCase name="testing with jest numbers two plus two" duration="0"/>
    <testCase name="testing with jest numbers adding floating point numbers" duration="1"/>
    <testCase name="testing with jest strings there is no I in team" duration="1"/>
    <testCase name="testing with jest strings but there is a &quot;stop&quot; in Christoph" duration="1"/>
    <testCase name="testing with jest arrays and iterables the shopping list has beer on it" duration="1"/>
    <testCase name="testing with jest exceptions compiling android goes as expected" duration="2"/>
    <testCase name="testing with jest callbacks the data is peanut butter" duration="104"/>
    <testCase name="testing with jest promises/resolve/reject/catch/async/await the data is peanut butter, then" duration="106"/>
    <testCase name="testing with jest promises/resolve/reject/catch/async/await the fetch fails with an error, catch" duration="102"/>
    <testCase name="testing with jest promises/resolve/reject/catch/async/await the data is peanut butter, resolves" duration="101"/>
    <testCase name="testing with jest promises/resolve/reject/catch/async/await the fetch fails with an error, rejects" duration="104"/>
    <testCase name="testing with jest promises/resolve/reject/catch/async/await the data is peanut butter" duration="104"/>
    <testCase name="testing with jest promises/resolve/reject/catch/async/await the fetch fails with an error" duration="102"/>
    <testCase name="testing with jest promises/resolve/reject/catch/async/await the data is peanut butter" duration="102"/>
    <testCase name="testing with jest promises/resolve/reject/catch/async/await the fetch fails with an error" duration="102"/>
    <testCase name="testing with jest should check truth" duration="1"/>
  </file>
</testExecutions>